test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/XPlatAddPkgTests.cs

```diff
@@ -745,6 +746,123 @@ public async Task AddPkg_ConditionalAddWithUserInputFramework_Success(string pac
             }
         }
 
+        [Fact]
+        public async Task AddPkg_WithAdditionalSourceFeed_NoVersionSpecified_Success()
+        {
+            using (var pathContext = new SimpleTestPathContext())
+            {
+                var projectFrameworks = "net472";
+                var packageFrameworks = "net472; netcoreapp2.0";
+                var projectA = XPlatTestUtils.CreateProject(ProjectName, pathContext, projectFrameworks);
+                var packageX = XPlatTestUtils.CreatePackage(frameworkString: packageFrameworks);
+                var packageXPath = Path.Combine(pathContext.WorkingDirectory, "packageX");
+                var mySourcePath = Path.Combine(pathContext.WorkingDirectory, "MySource");
+
+                Directory.CreateDirectory(mySourcePath);
+
+                // Generate Package
+                await SimpleTestPackageUtility.CreateFolderFeedV3Async(
+                    packageXPath,
+                    PackageSaveMode.Defaultv3,
+                    packageX);
+
+                string originalPackageXNupkg = Path.Combine(packageXPath, packageX.Id, packageX.Version, $"{packageX.Id}.{ packageX.Version}.nupkg");
+                string mySourcePackageXNupkg = Path.Combine(mySourcePath, $"{packageX.Id}.{ packageX.Version}.nupkg");
+
+                if (!File.Exists(originalPackageXNupkg))
+                {
+                    // On linux it's created in different place.
+                    string[] nupkgFiles = Directory.GetFiles(packageXPath, $"*.nupkg", SearchOption.AllDirectories);
+                    Assert.True(nupkgFiles.Count() == 1);
+                    originalPackageXNupkg = nupkgFiles[0];
+                }
+
+                // We're testing case we have only Nupkg file created at top.
+                File.Move(originalPackageXNupkg, mySourcePackageXNupkg);
```

Comment:
you created a v3 feed, which is consisted of more than just 1 file. Moving 1 file creates an inconsistent state in the original location.

Reply:
look up v3 feeds in the sanity guide in the one note.

Reply:
What I'm trying to do is just get 1 Nupkg file, actually I don't need v3 feed.
Because `dotnet pack --include-symbols --output ..\Mysource` command just create 1 Nugkg file nothing else.
I saw [`Util.CreateTestPackage`](https://github.com/NuGet/NuGet.Client/blob/223189bbc21e8259e6771d363e69d37d9b10e693/test/NuGet.Clients.Tests/NuGet.CommandLine.Test/Util.cs#L223) does actually that one but it's in different assembly and copying this method or referencing that assembly bring a lot of things.
So it's just work around to get nupkg file.

Reply:
If you need V2, then use https://github.com/NuGet/NuGet.Client/blob/223189bbc21e8259e6771d363e69d37d9b10e693/test/TestUtilities/Test.Utility/SimpleTestSetup/SimpleTestPackageUtility.cs#L428-L433.

There are a lot of helper APIs we've built over time.

Reply:
Ok. Done. I'm still discovering them.